Add warning about 0.16.x and earlier examples in quick start guide (#2244)
bevyengine/bevy-website: content/learn/quick-start/getting-started/_index.md

    cargo run --example breakout
    ```

{% callout(type="warning") %}

> Since Rust `1.89+` and when you use Bevy `0.16.x` and earlier versions the examples will not run and instead error. For more info on *why* see the issue: [bevyengine/bevy#20475 "0.16 fails to build breakout example due to bevy_mikktspace not compiling on Rust 1.89"](<https://github.com/bevyengine/bevy/issues/20475>)

Author: ```suggestion
When using Rust `1.89+` and Bevy `0.16.x or earlier` the examples will not run and instead error. For more info on *why* see the issue: [bevyengine/bevy#20475 "0.16 fails to build breakout example due to bevy_mikktspace not compiling on Rust 1.89"](<https://github.com/bevyengine/bevy/issues/20475>)
```
